lightning_pose/utils/predictions.py

```diff
@@ -304,7 +323,12 @@ def predict_dataset(
 
     pred_handler = PredictionHandler(cfg=cfg, data_module=data_module, video_file=None)
     labeled_preds_df = pred_handler(preds=labeled_preds)
-    labeled_preds_df.to_csv(preds_file)
+    if isinstance(labeled_preds_df, dict):
+        for view_name, df in labeled_preds_df.items():
+            view_name = "_" + view_name + ".csv"
+            df.to_csv(preds_file.replace(".csv", view_name))
```

Comment:
instead of view_name = "_" + ...

can do

df.to_csv(preds_file.replace(".csv", f"_{view_name}.csv"))